StyleCop.Analyzers/StyleCop.Analyzers/Helpers/NamedTypeHelpers.cs

```diff
@@ -145,11 +145,6 @@ internal static bool IsPartialDeclaration(MemberDeclarationSyntax declaration)
         /// <returns>true if the member is implementing an interface member, otherwise false.</returns>
         internal static bool IsImplementingAnInterfaceMember(ISymbol memberSymbol)
         {
-            if (memberSymbol.IsStatic)
```

Comment:
I assume this was just an optimization that can no longer remain, at least not in this simple form